website/static/js/comment.js

```diff
@@ -291,6 +511,50 @@ var CommentModel = function(data, $parent, $root) {
         self.author = self.$root.author;
     }
 
+    self.editableContent = ko.computed(function() {
+        var content = self.content();
+        if (!content) {
+            content = '';
+        }
+        var regex = /\[(@|\+)(.*?)\]\(\/([a-z\d]{5})\/\)/;
+        var matches = content.match(/\[(@|\+)(.*?)\]\(\/([a-z\d]{5})\/\)/g);
```

Comment:
A few thoughts here:

 - If we use constructor notation instead of a RegEx literal, we could define the pattern in one place, and instantiate the RegEx both with and without the `g` flag.
 - Why do we need a separate RegEx without the `g` flag (the `regex` variable here)? Wouldn't the `g` version also work for the second regex match in our algorithm here (the one inside the loop)?
 - Do we really need two RegExes here? We can't make this work with a single RegEx?